src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -84,6 +84,14 @@ private bool IsIssue93860Fixed()
 
 				if (assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>() is AssemblyInformationalVersionAttribute aiva)
 				{
+#if __IOS__ || __CATALYST__
```

Comment:
the Microsoft define is `__MACCATALYST__` but this (used to) works because `__MACCATALYST__` also define `__IOS__`.

Reply:
Indeed, still keeping it this way to make it obvious for the use case.